Add initial E2E tests for packaged app using WebdriverIO

lucas-koehler opened the pull request:
Add standalone WebdriverIO tests that test the packaged app.
The tests are run with mocha. Two tests are added:
1. Simply test the application title
2. Test that plugins are installed and recognized as builtin.

Note: The tests are Linux only

lucas-koehler committed:
Add initial E2E tests for packaged app using WebdriverIO

Add standalone WebdriverIO tests that test the packaged app.
The tests are run with mocha. Two tests are added:
1. Simply test the application title
2. Test that plugins are installed and recognized as builtin.

jfaltermeier commented:
Thanks, looks good. One issue I had initially was that I forgot to run `yarn package` besides `yarn` before running the tests. So maybe we should document the steps to execute the tests as well.

lucas-koehler committed:
Add documentation how to run E2E tests

lucas-koehler committed:
Enable E2E tests on Windows

lucas-koehler committed:
Enable E2E tests on MacOs

- Specify path to MacOs product executable
- most keyboard shortcuts on Mac platform use Command key instead of Control
- Puppeteer calls the Command key "Meta" -> on Mac map Control to Meta

lucas-koehler commented:
@jfaltermeier Tests now should work on Linux, Windows, and MacOS. I did not change anything in the initial commit but just rebased.
